lib/vanity/experiment/ab_test.rb

```diff
@@ -25,6 +25,56 @@ def initialize(*args)
         super
         @score_method = DEFAULT_SCORE_METHOD
         @use_probabilities = nil
+        @is_default_set = false
+      end
+        
+      # -- Default --
+
+      # Call this method once to set a default alternative. Call without 
+      # arguments to obtain the current default.
+      #
+      # @example Set the default alternative
+      #   ab_test "Background color" do
+      #     alternatives "red", "blue", "orange"
+      #     default "red"
+      #   end
+      # @example Get the default alternative
+      #   assert experiment(:background_color).default == "red"
+      # TODO document default choice if not explicitly chosen (first alternative specified)
```

Comment:
Let's just document this default at the top of this method - and then we can remove this TODO!